thirdparty/rtf2html/fmt_opts.h

```diff
@@ -19,11 +19,7 @@ struct color {
    {
       return !(*this==clr);
    }
-   color &operator=(const color &clr)
```

Comment:
You should not do any changes in thirdparty code

Reply:
Yeah I thought not, but uh, figured I'd learn on the job

Reply:
Fixing now

Reply:
Rather disable that warning for that part of the codebase